README.md

```diff
@@ -15,14 +15,39 @@ Key features:
 
 ## Installation
 
-1. Download the latest release from the [releases page](https://github.com/mattermost/mattermost-plugin-content-moderation/releases)
-2. Upload the plugin to your Mattermost instance via System Console > Plugin Management
-3. Enable the plugin
-4. Configure the plugin with your moderation backend and settings
+1. Download the latest release from the [releases page](https://github.com/mattermost/mattermost-plugin-content-moderation/releases).
+2. Upload the plugin to your Mattermost instance via **System Console > Plugins > Plugin Management**.
+3. Enable the plugin.
+4. Configure the plugin with your moderation backend and settings.
 
 ## Agents Plugin Setup
 
-To use the Agents Plugin as your moderation backend, install and configure the Mattermost Agents Plugin with an agent that has "Enable Tools" disabled and is accessible to all users. We recommend using Mistral as the LLM model for content moderation tasks.
+To use the Mattermost Agents Plugin as your moderation backend, install and configure the Mattermost Agents Plugin with an agent that has **Enable Tools** disabled and is accessible to all users. We recommend using Mistral as the LLM model for content moderation tasks.
```

Comment:
I'm not sure we want to go as far as recommending Mistral.  I prefer to categorize it as a lower cost option.